src/components/main/Header/Header.js

```diff
@@ -22,7 +22,15 @@ class Header extends Component {
                     <h1>{process.env.REACT_APP_TITLE}</h1>
                 </Col>
                 <Col lg={2} className='text-right'>
-                  <small>Version: {process.env.REACT_APP_VERSION}</small>
+                  <p>Version: {process.env.REACT_APP_VERSION}</p>
+                  <p>
+                    <a
+                      href="https://data.pacificclimate.org/portal/docs/"
```

Comment:
Should this URL be configurable or relative?

Reply:
Yes, plus text.